seed/utils/search.py

```diff
@@ -287,7 +288,7 @@ def _build_extra_data_annotations(column_name: str, data_type: str) -> tuple[str
         annotations.update({
             final_field_name: Cast(cleaned_field_name, output_field=models.IntegerField())
         })
-    elif data_type in ['number', 'float', 'area', 'eui']:
+    elif data_type in ['number', 'float', 'area', 'eui', 'ghg', 'ghg_intensity']:
```

Comment:
This may resolve some other unexpected issues with casting data via the search.py method.